.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "modules/PX4Firmware"]
 	path = modules/PX4Firmware
-	url = git://github.com/ArduPilot/PX4Firmware.git
+	url = https://github.com/crashmatt/Firmware.git
```

Comment:
The PX4 project appears to pull submodules from fork/branch.  I might have been fooling myself.

Next step is to get those mavlnk messages fixed.

Considering the volume of real world fixed wing users I am tending to think that ArduPlane is best bet.